client/src/components/employer-component/contacts-information-card/index.jsx

```diff
@@ -0,0 +1,407 @@
+  };
+
+  const closeAdd = () => {
+    setConfirmCancelAddDialog(false);
+  };
+
+  const cancelAddClose = () => {
+    setConfirmCancelAddDialog(false);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    setIsLoading(true);
+
+    // const modifiedEmployerInfo = {
+    //   id: employer.id,
+    //   name: employerName,
+    //   phone_number: employerPhoneNumber,
+    //   fax: employerFax,
+    //   email: employerEmail,
+    //   website: employerWebsite,
+    //   naics_code: employerNAICSCode,
+    //   address: employerAddress,
+    // };
+
+    // try {
+    //   const response = await modifyEmployerInfo(modifiedEmployerInfo);
+    //   console.warn(employer);
+    //   console.warn(response);
+    //   if (response.ok) {
+    //     setSnackBarMessage("Job lead updated successfully.");
+    //     setEditable(false);
+    //   } else {
+    //     setSnackBarMessage("Failed to update job lead.");
+    //   }
+    // } catch (error) {
+    //   setErrorObj(error);
+    //   setSnackBarMessage("An error occurred.");
+    // } finally {
+    //   setIsLoading(false);
+    //   setConfirmEditDialog(false);
+    // }
+  };
+
+  const leftButtonClick = () => {
+    if (contactPage > 1) {
+      const newContactPage = contactPage - 1;
+      setContactPage(newContactPage);
+      setName(contacts[newContactPage - 1].name);
+      setJobTitle(contacts[newContactPage - 1].jobTitle);
+      setEmail(contacts[newContactPage - 1].email);
+      setPhoneNumber(contacts[newContactPage - 1].phoneNumber);
+      setAlternativePhoneNumber(
+        contacts[newContactPage - 1].alternatePhoneNumber,
+      );
+    }
+  };
+
+  const rightButtonClick = () => {
+    if (contactPage < contacts.length) {
+      const newContactPage = contactPage + 1;
+      setContactPage(newContactPage);
+      setName(contacts[newContactPage - 1].name);
+      setJobTitle(contacts[newContactPage - 1].jobTitle);
+      setEmail(contacts[newContactPage - 1].email);
+      setPhoneNumber(contacts[newContactPage - 1].phoneNumber);
+      setAlternativePhoneNumber(
+        contacts[newContactPage - 1].alternatePhoneNumber,
+      );
+    }
+  };
+
+  useEffect(() => {
+    setName(contacts[0] ? contacts[0].name : "");
+    setJobTitle(contacts[0] ? contacts[0].jobTitle : "");
+    setEmail(contacts[0] ? contacts[0].email : "");
+    setPhoneNumber(contacts[0] ? contacts[0].phoneNumber : "");
+    setAlternativePhoneNumber(
+      contacts[0] ? contacts[0].alternatePhoneNumber : "",
+    );
+  }, [contacts]);
+
+  const emailWrapper = (contactEmail) => {
+    return <a href={`mailto:${contactEmail}`}>{contactEmail}</a>;
+  };
+
+  const phoneNumberWrapper = (contactPhone) => {
+    return <a href={`tel:${contactPhone}`}>{contactPhone}</a>;
+  };
+  const alternativePhoneNumberWrapper = (contactAltPhone) => {
+    return <a href={`tel:${contactAltPhone}`}>{contactAltPhone}</a>;
+  };
+
+  return (
+    <Card
```

Comment:
Take a look at the edit-job-lead-form component seeing how the dividers are set up for the card.

This could clean up some of the spacing in the header of the card and center the text a bit more

Reply:
Hi Jordan! What do you mean by cleaning up the spacing and centering of the text?